.env.template

```diff
@@ -0,0 +1,3 @@
+REACT_APP_DHIS2_BASE_URL=http://localhost:8080
+REACT_APP_DHIS2_API_VERSION=30
```

Comment:
If you have ejected and control, then config should be read from `DHIS2_HOME/config.json` like other apps for consistency.